tools/sandbox.py

```diff
@@ -2,12 +2,15 @@
 # Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
 
+# pylint:disable=invalid-name
```

Comment:
what is this for?

Reply:
it probably wants everything to be wrapped inside a `if __name__ == '__main__':` block